docs/rfc/rfc-1.adoc

```diff
@@ -43,6 +43,42 @@ collects tBTC fees it uses those fees to buy T tokens back for its treasury.
 
 == In Depth
 
+[[group-size]]
+=== Group Size
+There are three main governable parameters that we need to choose:
+* The number of signers required to sign messages `threshold`
+* The number of group members required for a <<heartbeat,heartbeat>> to be considered "strong". `heartbeat`
+* The minimum number of members we're allowed to drop down to during the DKG
+  group formation re-try period. `dkg`
+* The total size of the group `groupSize`
```

Comment:
This is not rendered as a list, see the [preview](https://github.com/keep-network/tbtc-v2/blob/bed5cae0d1970924a658f6e3f47fadefd382aa39/docs/rfc/rfc-1.adoc#group-size)

Reply:
Should be good now:
![image](https://user-images.githubusercontent.com/1045160/131913755-4503f936-0ac1-416c-8aee-efad6d4a548e.png)

8bd92e7